examples/mallocMC_example01.cu

```diff
@@ -122,10 +122,10 @@ void run()
   std::vector<int> array_sums(block*grid,0);
 
   // create arrays of arrays on the device
-  createArrays<<<1,1>>>(grid,block);
+  createArrays<<<1,1>>>(grid,block, mMC.devAllocator);
```

Comment:
Yes this looks more save than use a pointer. Than the handle is something like an interface for the allocator.